datawig/simple_imputer.py

```diff
@@ -586,3 +603,65 @@ def load_hpo_model(self, hpo_name: int = None):
 
         self.imputer = Imputer.load(self.output_path)
 
+    def check_for_label_shift(self,
+                              target_data: pd.DataFrame) -> tuple:
+
+        """
+        Detect label shift in the validation data
+
+        :param test_data: data frame that contains labels
+        :param target_data: unlabelled data for which predictions are to be generated
+
+        :return: tuple of label weight dict
+        """
+
+        # labels need to be sorted consistently for computation of confusion matrices etc.
+        labels = sorted(self.imputer.label_encoders[0].idx_to_token.values())
+
+        # generate predictions for test and target data
+        imputed_target = self.predict(target_data)
+        # imputed_test = self.predict(test_data)
+
+        # invert normalized empirical confusion matrix for test data
+        ids = dict((label, index) for (index, label) in enumerate(labels))
+        confusion_matrix_from_file = self.load_metrics()['confusion_matrix']
+        confusion_test = np.empty([len(confusion_matrix_from_file), len(confusion_matrix_from_file)])
+
+        for row in confusion_matrix_from_file:
```

Comment:
maybe put this in a helper method and comment it, its a bit confusing to read (probably only because I dont know the serialization format of the confusion matrix)